docs/userguide.rst

```diff
@@ -199,12 +199,14 @@ workflow workspace via the ``open`` command.
 .. code-block:: console
 
    $ reana-client open -w myanalysis.1 jupyter
-   http://reana.cern.ch:30080/7cd4d23e-48d1-4f7f-8a3c-3a6d256fb8bc?token=P-IkL_7w25IDHhes8I7DtICWLNQm2WAZ9gkoKC2vq10
```

Comment:
You can add some introductory scenario phrase such as:

While your analysis workflows are running, you may want to open interactive session processes on the workspace, such as Jupyter notebooks, via the ``open`` command.  This will allow to quickly explore the generated data while the analysis is in progress, or even run your analyses from within the notebook environment spawned on the remote containerised platform..